src/js/utils/css.js

```diff
@@ -72,49 +53,22 @@ define([
             elements = [elements];
         }
 
+        var style;
         var cssRules = {};
-        _updateStyleAttributes(cssRules, styles);
-
-        if (_cssBlock !== null && !immediate) {
-            if (!elements.__cssRules) {
-                elements.__cssRules = {};
-            }
-            _.extend(elements.__cssRules, cssRules);
-            if (_.indexOf(_cssBlock.elements, elements) < 0) {
-                _cssBlock.elements.push(elements);
-            }
-            // finish this later
-            return;
-        }
-        _updateElementsStyle(elements, cssRules);
-    };
-
-    var _block = function (id) {
-        // use id so that the first blocker gets to unblock
-        if (_cssBlock === null) {
-            // console.time('block_'+id);
-            _cssBlock = {
-                id: id,
-                styleSheets: {},
-                elements: []
-            };
+        for (style in styles) {
```

Comment:
Last I checked, for..in loops are considered a little slow, and prone to mis-use. People tend to prefer normal for loops https://jsperf.com/for-loop-vs-for-in-loop/4